tgui/packages/tgui/interfaces/Airlock.js

```diff
@@ -0,0 +1,93 @@
+import { useBackend } from '../backend';
+import { Box, Button, ProgressBar, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+// The minimum and maximum value on the progress bar before it overflow.
+const MIN_BAR_VALUE = 0;
+const MAX_BAR_VALUE = 1013;
+
+
+/* Definition of pressure threshold. 95 - 110 is considered good. 80 - 95 and 110 - 120 is considered average.
+Everything else bad.
+*/
+
+const getStatusColor = val => {
+  let statusColor = 'good';
+  const BAD_LOWER_THRESHOLD = 80;
+  const AVG_LOWER_THRESHOLD = 95;
+  const AVG_UPPER_THRESHOLD = 110;
+  const BAD_UPPER_THRESHOLD = 120;
+  if (val < BAD_LOWER_THRESHOLD) {
+    statusColor = 'bad';
+  } else if (val < AVG_LOWER_THRESHOLD) {
+    statusColor = 'average';
+  } else if (val > AVG_UPPER_THRESHOLD) {
+    statusColor = 'average';
+  } else if (val > BAD_UPPER_THRESHOLD) {
+    statusColor = 'bad';
+  }
+  return statusColor;
+};
+
+export const Airlock = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    chamber_pressure,
+    exterior_status,
+    interior_status,
+    processing,
+  } = data;
+  return (
+    <Window>
+      <Window.Content>
+        <Section title="Information">
+          <LabeledList>
+            <LabeledList.Item label="Chamber Pressure">
+              <ProgressBar
+                color={getStatusColor(data.chamber_pressure)}
+                value={data.chamber_pressure}
+                minValue={MIN_BAR_VALUE}
+                maxValue={MAX_BAR_VALUE}>
+                {data.chamber_pressure} kPa
```

Comment:
```suggestion
                color={getStatusColor(chamber_pressure)}
                value={chamber_pressure}
                minValue={MIN_BAR_VALUE}
                maxValue={MAX_BAR_VALUE}>
                {chamber_pressure} kPa
```
If youre defining these off of data at line 34-39, you dont need to include `data.` in these statements